exporter/modules.go

```diff
@@ -0,0 +1,51 @@
+package exporter
+
+import (
+	"strings"
+
+	"github.com/gomodule/redigo/redis"
+	"github.com/prometheus/client_golang/prometheus"
+	log "github.com/sirupsen/logrus"
+)
+
+func (e *Exporter) extractModulesMetrics(ch chan<- prometheus.Metric, c redis.Conn) {
+	info, err := redis.String(doRedisCmd(c, "INFO", "MODULES"))
+	if err != nil {
+		log.Errorf("extractSearchMetrics() err: %s", err)
+		return
+	}
+
+	lines := strings.Split(info, "\r\n")
+	for _, line := range lines {
+		log.Debugf("info: %s", line)
+
+		split := strings.Split(line, ":")
+		switch {
+		case split[0] == "module":
+			module := strings.Split(split[1], ",")
```

Comment:
same, can we wrap this with a len() check?

`if module := split(); len(module) > ...` should do the trick

Reply:
Done, added.

Reply:
👍